tests/lib/then_steps.py

```diff
@@ -210,6 +210,22 @@ def journal_should_not_exist(config_on_disk, should_or_should_not):
         )
 
 
+@then(parse('the journal "{journal_name}" directory {should_or_should_not} exist'))
+def directory_should_not_exist(config_on_disk, should_or_should_not, journal_name):
+    scoped_config = scope_config(config_on_disk, journal_name)
+    expected_path = scoped_config["journal"]
+
+    dir_exists = os.path.isdir(expected_path)
+    if should_or_should_not == "should":
+        assert dir_exists
+    elif should_or_should_not == "should not":
+        assert not dir_exists
```

Comment:
Could you use the parse_should_or_should_not(should_or_should_not) method here? See other methods in this file like config_var_on_disk for an example.